torch/fx/proxy.py

```diff
@@ -154,7 +154,7 @@ def __torch_function__(self, orig_method, types, args=None, kwargs=None):
             return self.tracer.create_proxy('call_method', orig_method.__name__, args, kwargs)
         else:
             return self.tracer.create_proxy('call_function', orig_method, args, kwargs,
-                                            name=self.tracer.graph._name(orig_method.__name__))
+                                            name=orig_method.__name__)
```

Comment:
With the other suggestion to move the snake case stuff, you probably still want to call _target_to_str here so that you get a reasonable snake_case name.